.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+# changes to documentation
+"area/documentation": doc/**/*
+
+# changes to the core lib package
+"area/lib": ./*.go
+
```

Comment:
What about one for the Cobra Generator itself?

```
# changes to the cobra generator
"area/cobra-generator": cobra/**/*
```

Reply:
I didn't see that label in the current set.

Maybe we have a enhancement PR to add more labels and matchers?
@jpmcb